pkg/apis/topology/v1alpha1/generated.proto

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// This package imports things required by build scripts, to force `go mod` to see them as dependencies
+
+// This file was autogenerated by go-to-protobuf. Do not edit it manually!
+
+syntax = "proto2";
```

Comment:
Why proto2 here and not proto3?

Reply:
I think we don't have an option: https://github.com/kubernetes/code-generator/blob/ee3ba0fad71e3afe882f946036856ada0a0fc603/cmd/go-to-protobuf/protobuf/generator.go#L740
https://github.com/kubernetes/code-generator/blob/ee3ba0fad71e3afe882f946036856ada0a0fc603/cmd/go-to-protobuf/protobuf/generator.go#L439

Reply:
Yup, that's the reason. No alternatives here. But shouldn't cause any issues as proto2 can be included in proto3 just fine.

Reply:
Ack, thanks to both for the clarification.